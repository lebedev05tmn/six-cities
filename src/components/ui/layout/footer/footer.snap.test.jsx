import React from "react";
import {render} from "@testing-library/react";
import {BrowserRouter} from "react-router-dom";
import Footer from "./footer";

test(`Should Footer renders correctly`, () => {
  const {container} = render(
    <BrowserRouter>
      <Footer />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
