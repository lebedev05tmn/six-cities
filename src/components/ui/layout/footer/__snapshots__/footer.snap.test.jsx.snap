// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Footer renders correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <a
      class="footer__logo-link"
      href="/"
    >
      <img
        alt="6 cities logo"
        class="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </a>
  </footer>
</div>
`;
